ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentFlows.css

```diff
@@ -0,0 +1,3 @@
+.pf-c-table__toggle {
+    padding: 0 !important
+}
```

Comment:
oh is there extra padding otherwise?

Reply:
Yea this is the difference

<img width="1552" alt="Screenshot 2022-11-04 at 9 19 13 AM" src="https://user-images.githubusercontent.com/4805485/200025143-02e9cb9a-d3bf-4d04-a854-2f1adfa761c7.png">
<img width="1552" alt="Screenshot 2022-11-04 at 9 19 21 AM" src="https://user-images.githubusercontent.com/4805485/200025160-06cff434-9a6b-44bd-81ab-a8fd9ddcffe8.png">